test/run_tests.sh

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
 docker build -t local/test/sdcli .
-docker build -t local/test/sdclitests -f test.Dockerfile .
+docker build -t local/test/sdclitests test
```

Comment:
question: does this know to run `test.Dockerfile` when you give it the `test` argument? I thought the -f would be needed here

Reply:
Nope, the first argument is path to context. Dockerfile is picked up automatically in the root of context by default